R_PACKAGES.md

```diff
@@ -0,0 +1,197 @@
+# R Packages for Open Systems Pharmacology
+
+The Open Systems Pharmacology (OSP) ecosystem provides a comprehensive suite of R packages that enable scripted workflows with models developed in PK-Sim® and MoBi®. These packages allow users to perform various pharmacokinetic and pharmacodynamic analyses, create standardized reports, and develop advanced modeling workflows.
+
+## Core Packages
+
+### ospsuite
+
+**Repository:** [OSPSuite-R](https://github.com/Open-Systems-Pharmacology/OSPSuite-R)  
+**Documentation:** [https://www.open-systems-pharmacology.org/OSPSuite-R/](https://www.open-systems-pharmacology.org/OSPSuite-R/)  
+**Description:** The core R package that provides functionality for loading, manipulating, and simulating models created in PK-Sim® and MoBi®. It offers extended workflows including parameter sensitivity analysis, PK-parameter calculation, population simulations, and individual creation.
+
+**Key Features:**
+- Load and run simulations from PK-Sim® and MoBi®
+- Parameter and molecule start value manipulation
+- Population simulation capabilities
+- PK analysis and sensitivity analysis
+- Data import/export functionality
+- Unit conversion and dimension handling
+
+### tlf
```

Comment:
The `tlf` package should be listed under "Reporting and Visualization", after the `ospsuite.plots` package.